flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/motion/supervised/Supervisable.java

```diff
@@ -0,0 +1,7 @@
+package com.flash3388.flashlib.robot.motion.supervised;
+
+import com.flash3388.flashlib.control.Stoppable;
+
+public interface Supervisable extends Stoppable {

```

Comment:
`Supervisable` isn't the best name for it. I understand where it comes from, but saying _supervision_ implies something completely different.

Maybe something around `Limit`?